lib/geordi/commands/rake.rb

```diff
@@ -12,7 +12,7 @@
 def rake(*args)
   %w[development test cucumber].each do |env| # update long_desc when changing this
     if File.exist? "config/environments/#{env}.rb"
-      call = %w[bundle exec rake] + args + ["RAILS_ENV=#{env}"]
+      call = Util.binstub('rake') + args + ["RAILS_ENV=#{env}"]
```

Comment:
I doubt this works as expected, since you're adding String + Array + Array. You should wrap the first summand with brackets `[]`.